db/migrations/000096-01-alter-table-shantytowns.js

```diff
@@ -0,0 +1,125 @@
+const regularAccess = 'L\'accès à l\'électricité est régulier.';
+const irregularAccess = 'L\'accès à l\'électricité est irrégulier.';
+
+module.exports = {
+    up: (queryInterface, Sequelize) => queryInterface.sequelize.transaction(
+        // Update Oui (acces régulier)
+        transaction => queryInterface.sequelize.query(
+            'UPDATE shantytowns SET electricity_comments = :value WHERE fk_electricity_type = 4',
+            {
+                transaction,
+                replacements: { value: regularAccess },
+            },
+        )
+            .then(() => queryInterface.sequelize.query(
+                'UPDATE "ShantytownHistories" SET electricity_comments = :value where fk_electricity_type = 4',
+                {
+                    transaction,
+                    replacements: { value: regularAccess },
+                },
+
+            ))
+
+            // Update Oui (acces irrégulier)
+            .then(() => queryInterface.sequelize.query(
+                'UPDATE shantytowns SET electricity_comments = :value where fk_electricity_type = 5',
+                {
+                    transaction,
+                    replacements: { value: irregularAccess },
+                },
+            ))
+            .then(() => queryInterface.sequelize.query(
+                'UPDATE "ShantytownHistories" SET electricity_comments = :value where fk_electricity_type = 5',
+                {
+                    transaction,
+                    replacements: { value: irregularAccess },
+                },
+            ))
+            // Update Reference to Oui
+            .then(() => queryInterface.sequelize.query(
+                'UPDATE shantytowns SET fk_electricity_type = 3 where fk_electricity_type = 4 OR fk_electricity_type = 5',
+                {
+                    transaction,
+                },
+            ))
+            .then(() => queryInterface.sequelize.query(
+                'UPDATE "ShantytownHistories" SET fk_electricity_type = 3 where fk_electricity_type = 4 OR fk_electricity_type = 5',
```

Comment:
Autre façon de l'écrire, plus courte, : `WHERE fk_electricity_type IN (4, 5)`

Reply:
C'est corrigé